client/src/ui/molecules/theme-toggle/index.tsx

```diff
@@ -28,7 +28,7 @@ export function ThemeToggle() {
 
   return (
     <div className="theme-toggle">
-      <Icon name="theme"></Icon>
+      {dark ? <Icon name="moon-fill"></Icon> : <Icon name="sun-fill"></Icon>}
```

Comment:
Can we write this as follows, to avoid duplication?

```suggestion
      <Icon name={dark ? "moon-fill" : "sun-fill"} />
```